src/main/java/nextstep/subway/line/domain/Sections.java

```diff
@@ -0,0 +1,68 @@
+package nextstep.subway.line.domain;
+
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.Comparator;
+import java.util.List;
+import java.util.stream.Collectors;
+
+import javax.persistence.CascadeType;
+import javax.persistence.Embeddable;
+import javax.persistence.OneToMany;
+
+import nextstep.subway.station.domain.Station;
+
+@Embeddable
+public class Sections {
+
+	@OneToMany(mappedBy = "line", cascade = CascadeType.ALL)
+	private List<Section> sections;
```

Comment:
OneToMany 연관관계의 경우 명시적 초기화를 해주는 것이 일반적인 컨벤션입니다 :)

```suggestion
	@OneToMany(mappedBy = "line", cascade = CascadeType.ALL)
	private List<Section> sections = new ArrayList<>(sections);
```

Reply:
명시적 초기화. 감사합니다. JPA의 일반적 컨벤션을 다시 숙지해봐야겠네요